testsuite/Makefile

```diff
@@ -253,7 +256,7 @@ one: lib tools
 	@case "$(words $(DIR) $(LIST) $(TEST))" in \
    0) echo 'No value set for variable DIR, LIST or TEST'>&2; exit 1;; \
    1) exit 0;; \
-   *) echo 'Please specify just one of DIR, LIST or TEST'>&2; exit 1;; \
+   *) echo 'Please specify exactly one of DIR, LIST or TEST'>&2; exit 1;; \
```

Comment:
That has an explicit error two lines earlier, though!

Reply:
It does, but I think any error message you get should be clear in isolation – why make somebody round-trip through two errors instead of one?  It's also possible that the `0` and `*` cases could share an error message, too...